### PR TITLE
feat: tikv get

### PR DESCRIPTION
**Pull request by YikSanChan**

### CODE REVIEW

**Review by lianxmfor — APPROVED** [review]
lgtm

File: internal/database/online/tikv/get.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	}
+	if rowMap, ok := res[opt.EntityKey]; !ok {
+		return nil, fmt.Errorf("entity key %s not found", opt.EntityKey)
+	} else {
+		return rowMap, nil
+	}
+}
+
+func (db *DB) MultiGet(ctx context.Context, opt online.MultiGetOpt) (map[string]dbutil.RowMap, error) {
+	serializedRevisionID, err := redis.SerializeByValue(opt.RevisionID)
```
Comment (wfxr) [inline internal/database/online/tikv/get.go:37]: How about moving `redis.SerializeByValue` to a util package? We probably use these functions in most kv stores.

Comment (YikSanChan) [reply internal/database/online/tikv/get.go:37]: about to, though in separate PR

**Review by wfxr — APPROVED** [review]
LGTM

**Pull request merged**